sda-commons-dependencies/build.gradle

```diff
@@ -37,6 +37,8 @@ dependencies {
     // overall testing dependencies
     api 'org.awaitility:awaitility:4.3.0'
 
+    api "org.junit-pioneer:junit-pioneer:2.3.0"
```

Comment:
In the services I added this dependency, I had to do an exclusion, otherwise there were conflicts with die JUnit version: https://github.com/SDA-SE/rise-kvs-delivery-service/blob/main/build.gradle#L92-L94

I would have expected the managed dependency also to include something like this. Could you check if the transitive dependencies from junit-pioneer are in sync with managed junit dependencies

Additional info: In services we do strict dependency resolving: https://github.com/SDA-SE/rise-kvs-delivery-service/blob/main/build.gradle#L49-L55